stl/inc/optional

```diff
@@ -425,7 +426,7 @@ public:
         return _STD move(this->_Value);
     }
 
-    template <class _Ty2>
+    template <class _Ty2 = remove_cv_t<_Ty>>
     _NODISCARD constexpr remove_cv_t<_Ty> value_or(_Ty2&& _Right) const& {

```

Comment:
LWG-3424 wasn't completely implemented for `optional` - we should complete it now.